index.html

```diff
@@ -0,0 +1,230 @@
+                    <button class="recent-ruby-btn">Ruby</button>
+                </div>
+                <button class="recent-see-btn">See project</button>
+            </article>
+        </section>
+        <section class="section-3">
+            <div class="grid">
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
```

Comment:
- [ ] According to the project requirement, the `<div>` tag should be replaced with `<ul>` tag and the `<button>` be replaced with `<li>` tags. They are supposed to be list items. Below is the screenshot of the requirement.
![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/43528594/7cf65549-f65d-4c62-8d1a-26c09d5cd505)

Please ensure that you follow the requirement accordingly. :slightly_smiling_face: